bin/dev_stop.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [ -e tmp/pids/server.pid ]; then
+    kill -9 $(cat tmp/pids/server.pid)
+    rm tmp/pids/server.pid
+fi
+if [ -e tmp/pids/delayed_job.pid ]; then
+    kill -9 $(cat tmp/pids/delayed_job.pid)
+    rm tmp/pids/delayed_job.pid
+fi
+
```

Comment:
* take any shell scripting advice from me with a grain of salt.  but wouldn't it be better to use `kill` instead of a `kill -9`, to give things a chance to exit gracefully?  maybe echo the process IDs so that the user can easily check to see whether they're dead if they suspect something hung?
* also need to `rake jetty:stop`